packages/workbox-cache-expiration/test/browser/end-to-end.js

```diff
@@ -37,4 +41,40 @@ describe(`End to End Test of Cache Expiration`, function() {
 
     expect(keys).to.have.length(EXPECTED_CACHE_SIZE);
   });
+
+  it(`should work properly when a cache-first strategy + maxAgeSeconds is used, and responses lack a Date header`, async function() {
+    // See the comment later about a source of potential flakiness.
+    this.retries(2);
```

Comment:
you should probably set the timeout of this test case to 5 or 6 with `this.timeout(6 * 1000);`